dao/beacon/attestations.gen.go

```diff
@@ -0,0 +1,34 @@
+// Code generated by gorm.io/gen. DO NOT EDIT.
+// Code generated by gorm.io/gen. DO NOT EDIT.
+// Code generated by gorm.io/gen. DO NOT EDIT.
+
+package beacon
```

Comment:
all testnets of beacon are the same schema?

Reply:
yes